### PR TITLE
Add schedule payment service that is able to GET schedule payments

### PR DESCRIPTION
**Pull request by stijnkoopal**
Implements: https://doc.bunq.com/#/schedule-payment/List_all_SchedulePayment_for_User_MonetaryAccount

### CODE REVIEW

File: bunq/response_data.go [hunk 1]
```diff
@@ -63,6 +63,14 @@ type responseMasterCardActionGet struct {
 	Pagination Pagination `json:"Pagination"`
 }
 
+// ResponseSchedulePaymentGet The schedule payment response object.
+type ResponseSchedulePaymentGet struct {
+	Response []struct {
+		SchedulePayment SchedulePayment `json:"ScheduledPayment"`
```
Comment (OGKevin) [inline bunq/response_data.go]: Oh no, the endpoint is `schedule-payment` but the object is called `ScheduledPayment` with the `d`? Shall we just name all the structs `ScheduledPayment` with the `d` in this SDK?

**Review by OGKevin — COMMENTED** [review]
Small question comment

File: bunq/response_data.go (hunk 1 above)
Comment (stijnkoopal) [reply bunq/response_data.go]: Done for the schedule related stuff

**Review by OGKevin — APPROVED** [review]
Nice!

**Pull request merged**